python/kserve/kserve/model.py

```diff
@@ -139,19 +139,36 @@ def __init__(
         predictor_protocol: str = PredictorProtocol.REST_V1.value,
         predictor_use_ssl: bool = False,
         predictor_request_timeout_seconds: int = 600,
+        predictor_request_retries: int = 0,
+        predictor_health_check: bool = False,
     ):
         """The configuration for the http call to the predictor
 
         Args:
             predictor_host: The host name of the predictor
             predictor_protocol: The inference protocol used for predictor http call
             predictor_use_ssl: Enable using ssl for http connection to the predictor
-            predictor_request_timeout_seconds: The request timeout seconds for the predictor http call
+            predictor_request_timeout_seconds: The request timeout seconds for the predictor http call. Default is 600 seconds.
```

Comment:
isn't 600 too much for a inference request?

Reply:
I think it is because we don't want the request to be timed out here.

Reply:
I would agree, 60s probably is more reasonable for most of the real time inference requests